backend/middleware/auth.js

```diff
@@ -0,0 +1,15 @@
+const jwt = require("jsonwebtoken");
+
+module.exports = (req, res, next) => {
+  const token = req.header("token");
+  if (!token) return res.status(401).json({ message: "Auth Error" });
+
+  try {
+    const decoded = jwt.verify(token, "randomString"); // change this a stronger secret and add to .env file
```

Comment:
It there a reason this wasn't done in this PR?

Reply:
Oops! I'll move that out.